doc/devel/development_setup.rst

```diff
@@ -60,6 +60,16 @@ true for ``*.py`` files.  If you change the C-extension source (which might
 also happen if you change branches) you will have to re-run
 ``python -m pip install -ve .``
 
+Installing pre-commit hooks
```

Comment:
Would it be better to talk about using them (ie. `pre-commit run`) instead of installing them? Personally I prefer not having pre-commit editing my files automatically.

Reply:
How about adding a discussion of running them with out installing as hooks following this? 

Something like:

If you do not want these git hooks you can skip the `pre-commit install` step. You can still run the checks with `pre-commit run <list of files>` or `pre-commit run --all-files`

Reply:
👍 sounds good to me

Reply:
I think `pre-commit run ...` will still update/modify the files if that is what the hook is doing though? It is just if you `install` the hooks automatically get run on every commit without a user running the command explicitly.

:+1: to changing this section title to "Using pre-commit hooks" and expanding the text.